contracts/Flovatar.cdc

```diff
@@ -664,6 +702,15 @@ pub contract Flovatar: NonFungibleToken {
             let flovatarNFT = nft as! &Flovatar.NFT
             return flovatarNFT as &AnyResource{MetadataViews.Resolver}
         }
+
+        // This is what we are proposing for nested resources on how they should expose sub-Items
+        pub fun borrowViewResolverColelction(id: UInt64): &AnyResource{MetadataViews.ResolverCollection}? {
```

Comment:
@crash13override I noticed a typo here in "Colelction".

Reply:
thanks @Bamthelearner and @TraeRegan ,

Can you please get the name updated to that I can merge the PR and deploy it to testnet first and then mainnet?

Thanks!

Reply:
I belive this method is gone in the latest update?

Reply:
now that I am about to deploy the $DUST token it would be a good time to merge this PR if you're still interested. I noticed that this typo is still in the PR and not sure if the whole method should go away perhaps. Let me know if you want to take another look before I give a final review. 
Thanks!!!

Reply:
Should we set up a call and talk about the sub resources standard?